pages/linux/scan-build.md

```diff
@@ -0,0 +1,20 @@
+# scan-build
+
+> A command-line utility to run a static analyzer over a codebase as part of performing a regular build.
```

Comment:
Perhaps this will read better?
```suggestion
> Command-line utility to run a static analyzer over a codebase as part of performing a regular build.
```